canvas_site_deletion/views.py

```diff
@@ -120,7 +120,7 @@ def delete(request, pk):
                 )
 
             logger.info(f'Step 3/5: changing course {canvas_course_id} SIS ID to {canvas_course["sis_course_id"]}-deleted-{ts} and then deleting the course')
-            courses.update_course(SDK_CONTEXT, id=canvas_course_id, course_sis_course_id=f'{canvas_course["sis_course_id"]}-deleted-{ts}')
```

Comment:
> We identified new issues on unchanged lines of code. Navigate to the Amazon CodeGuru Reviewer console to view the recommendations to fix them.

@cmurtaugh So the [CodeGuru Reviewer Console](https://us-east-1.console.aws.amazon.com/codeguru/reviewer/codereviews/details/arn:aws:codeguru-reviewer:us-east-1:482956169056:association:c6f8f8f4-8b78-485c-82b0-7d9fd98e2c6f:code-review:PullRequest-GITHUB-canvas_account_admin_tools-437-136951babfe81413696649c70b547a95be650d32?region=us-east-1) calls out issue with changing the parameters in this use of `update_course` in the `delete` view of canvas_site_deletion. 

I was a bit confused on whether or not to update this one because it doesn't specifically bring out the `update_course` imported from Canvas SDK. So I guess it shouldn't be applied here since `update_course` comes out of `courses`?